acs-integration-tests/pom.xml

```diff
@@ -90,6 +90,9 @@
                 <artifactId>maven-failsafe-plugin</artifactId>
                 <version>2.18.1</version>
                 <configuration>
+                    <forkCount>2.0C</forkCount>
+                    <reuseForks>true</reuseForks>
+                    <parallel>classes</parallel>
```

Comment:
According to http://maven.apache.org/components/surefire/maven-failsafe-plugin/examples/fork-options-and-parallel-execution.html:
When using reuseForks=true and a forkCount value larger than one, test classes are handed over to the forked process one-by-one. Thus, parallel=classes would not change anything.